src/content/docs/victory-stack.md

```diff
@@ -12,10 +12,12 @@ scope:
 `VictoryStack` is a wrapper component that renders a given set of children in a stacked layout. Like other wrapper components, `VictoryStack` also reconciles the domain and layout for all its children, and coordinates animations and shared events.
 
 `VictoryStack` works with:
-[VictoryArea][], [VictoryBar][], [VictoryCandlestick][], [VictoryErrorBar][], [VictoryGroup][],[VictoryLine][], and [VictoryScatter][]
+[VictoryArea][], [VictoryBar][], [VictoryCandlestick][], [VictoryErrorBar][], [VictoryGroup][],[VictoryLine][], [VictoryScatter][], [VictoryHistogram][]
 
 `VictoryStack` _should not_ be used with [VictoryAxis][] children.
 
+`VictoryStack` when being used with `VictoryHistogram`s, all children must be `VictoryHistogram`s.
```

Comment:
This sentence is a bit confusing, maybe "`VictoryStack` does not support stacking `VictoryHistogram` with other Victory components." or  "When creating stacked histograms, `VictoryStack` must include _only_ `VictoryHistogram` children."